docs/authoring-components.md

```diff
@@ -725,6 +725,10 @@ Concretely:
   contains javascript, ensure that its component namespace is exported within
   `material-components-web`, and it is registered with `mdc-auto-init`. Lastly, remember to add it
   to `package.json` of `material-components-web`.
+- Ensure that the correct **commit subject** for the package is added to the
```

Comment:
Do you mean only if this package is added to `config.validate-commit-msg.scope.allowed` then its commit message will be checked? I tried to make a wrong commit message "feat(mdc-grid-list): Implement mdc-grid-list (#47)" but it still went through.